build/ci.go

```diff
@@ -215,6 +215,9 @@ func doInstall(cmdline []string) {
 	// Configure the build.
 	env := build.Env()
 	gobuild := tc.Go("build", buildFlags(env)...)
+	if len(*buildtags) != 0 {
```

Comment:
Could we add a comment here?

Reply:
I think it's has been described in `buildtags = flag.String("buildtags", "", "Tags for go build")`